### DISCUSSION TITLE
QuasarUI by Hafiz Javaid Iqbal

### DISCUSSION THREAD

**Discussion by jclaveau** [quasarframework/quasar#17707] [Show and Tell]
Looking for some Quasar UI extensions I found this https://www.quasarui.com

This seems misleading content as the name could mean it's from the official Quasar team and, by the way, is not free when Quasar relies on donations.

Isn't there any intellectual property issue here, regarding the name of this product?

**Comment by hafizjavaid** [comment]
Hafiz Javaid Iqbal here, @jclaveau  why do you think it's misleading

First of all I got this domain name idea  because we have same kind of components for tailwind ( tailwindui ), nuxt ( NuxtUI ).

You said "name could mean it's from the official Quasar team" I think you are totally wrong here

Even your discussion title is saying "QuasarUI by Hafiz Javaid Iqbal". Please focus on "By".

Did you found anywhere in my website where I mentioned it's from the Quasar team

**Reply by jclaveau** [reply]
- NuxtUI is developped by the Nuxt Team
- Tailwind UI is developped by Tailwind Labs Inc.
- They both possess the trademarks Nuxt and Tailwind
- I don't know if you've ever registered a trademark but the very basics is to not use a name too close to another one of the same activity area. Which is your case here

So, basically, you are behaving in a way which may be outside the law. 

This may not be a problem depending on what the Quasar Team consider but IMHO, the minimum is to ask them.
Please, take, in consideration that, for example, the quality of your product may impact their reputation, even if you have only good intentions.

So asking them if there is or not an intellectual property issue there is totally genuine.


Hoping you realize I have absolutely nothing against you and, I actually think your product is a very nice idea. But if you want to start your own business you will have to consider IP.

Good luck btw!
